app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -1,226 +1,561 @@
+                    const targetClusterCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'emscluster'
+                    );
+                    const sourceDatastoreCount = this.sourceCount(
+                      mapping.transformation_mapping_items,
+                      'storage'
+                    );
+                    const targetDatastoreCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'storage'
+                    );
+                    const sourceLanCount = this.sourceCount(
+                      mapping.transformation_mapping_items,
+                      'lan'
+                    );
+                    const targetLanCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'lan'
+                    );
+                    const associatedPlansCount =
+                      mapping.service_templates &&
+                      mapping.service_templates.length;
+
+                    const {
+                      targetClusters,
+                      targetDatastores,
+                      targetNetworks
+                    } = mapInfrastructureMappings(
+                      mapping.transformation_mapping_items,
+                      clusters,
+                      datastores,
+                      networks
+                    );
+
+                    return (
+                      <ListView.Item
+                        key={mapping.id}
+                        heading={mapping.name}
+                        description={
+                          <small>
+                            {__('Completed: ')}
+                            {formatDateTime(mapping.created_at)}
+                          </small>
+                        }
+                        stacked
+                        compoundExpand
+                        compoundExpanded={mapping.expanded}
+                        onCloseCompoundExpand={() => this.closeExpand(mapping)}
+                        additionalInfo={[
+                          <ListView.InfoItem key={0}>
+                            <ListView.Expand
+                              expanded={
+                                mapping.expanded && mapping.expandType === 0
+                              }
+                              toggleExpanded={() => {
+                                this.toggleExpand(mapping, 0);
+                              }}
+                            >
+                              <Icon type="pf" name="network" />
+                              <div className="mappings-expand-label-group">
+                                <div className="mappings-expand-label">
+                                  <strong>{sourceLanCount}</strong>
+                                  {` `}
+                                  {pluralize(sourceLanCount, 'Source Network')}
```

Comment:
The canonical gettext way of doing this would be `n_('Source Network', 'Source Networks', sourceLanCount)` ... except it's still not quite right, because the order is not necessarily  "# thingies" in all languages.

So... this would end up being `sprintf(n_('%d Source Network', '%d Source Networks', sourceLanCount), sourceLanCount)`.

(That, or

```
n_(`${sourceLanCount} Source Network`, `${sourceLanCount} Source Networks`)
```

once our i18n tooling supports template strings properly. (Unless it already does @mzazrivec ?)

Reply:
I don’t think the other 3rd party libraries are any more reliable than gettext (I’ve seen others mentioned too besides these). My vote is to stay consistent with gettext and if that’s the way this is achieved that’s fine, but it’s not reducing the original solution much. I still think on-Prem translation solutions end up on the translator. There is SaaS offerings for i18n (real time translation APIs) that are very reliable, but it doesn’t look like we’ll be investing in those.

Reply:
I don’t think the other 3rd party libraries are any more reliable than gettext (I’ve seen others mentioned too besides these). My vote is to stay consistent with gettext and if that’s the way this is achieved that’s fine, but it’s not reducing the original solution much. I still think on-Prem translation solutions end up on the translator. There is SaaS offerings for i18n (real time translation APIs) that are very reliable, but it doesn’t look like we’ll be investing in those.

Reply:
Was just looking for the one off solution for these cases (not pluralizing all)

Reply:
I don't want to render the count within the same block (because styling is different around the <strong> tags). So for now, I just renamed the method (I think my original intent was only for these simple scenarios), however I think it could be made more flexible this way...

Happy to revisit this in the future, but I think this is good enough for now.